src/js/video.js

```diff
@@ -33,7 +33,9 @@ import xhr from 'xhr';
 import Tech from './tech/tech.js';
 
 // HTML5 Element Shim for IE8
-if (typeof HTMLVideoElement === 'undefined') {
+if (typeof HTMLVideoElement === 'undefined' &&
+    window.document &&
+    window.document.createElement) {
```

Comment:
As I mentioned in the description, we can't use `document` because `global/document` tries to shim the DOM API in node.